rhessys/init/construct_basin.c

```diff
@@ -71,6 +72,7 @@ struct basin_object *construct_basin(
 	/*--------------------------------------------------------------*/
 	/*	Local function definition.									*/
 	/*--------------------------------------------------------------*/
+	param *readtag_worldfile(int *, FILE *,char *);	
```

Comment:
readtag_worldfile() should be declared once in params.h, which will obviate the need to do a local declaration everywhere it is used.